worlds/pokemon_rb/items.py

```diff
@@ -0,0 +1,176 @@
+    "Revive": ItemData(53, ItemClassification.filler, ["Consumables"]),
+    "Max Revive": ItemData(54, ItemClassification.filler, ["Consumables"]),
+    "Guard Spec": ItemData(55, ItemClassification.filler, ["Consumables", "Battle Items"]),
+    "Super Repel": ItemData(56, ItemClassification.filler, ["Consumables"]),
+    "Max Repel": ItemData(57, ItemClassification.filler, ["Consumables"]),
+    "Dire Hit": ItemData(58, ItemClassification.filler, ["Consumables", "Battle Items"]),
+    #"Coin": ItemData(59, ItemClassification.filler),
+    "Fresh Water": ItemData(60, ItemClassification.filler, ["Consumables"]),
+    "Soda Pop": ItemData(61, ItemClassification.filler, ["Consumables"]),
+    "Lemonade": ItemData(62, ItemClassification.filler, ["Consumables"]),
+    "S.S. Ticket": ItemData(63, ItemClassification.progression, ["Unique", "Key Items"]),
+    "Gold Teeth": ItemData(64, ItemClassification.progression, ["Unique", "Key Items"]),
+    "X Attack": ItemData(65, ItemClassification.filler, ["Consumables", "Battle Items"]),
+    "X Defend": ItemData(66, ItemClassification.filler, ["Consumables", "Battle Items"]),
+    "X Speed": ItemData(67, ItemClassification.filler, ["Consumables", "Battle Items"]),
+    "X Special": ItemData(68, ItemClassification.filler, ["Consumables", "Battle Items"]),
+    "Coin Case": ItemData(69, ItemClassification.progression_skip_balancing, ["Unique", "Key Items"]),
+    "Oak's Parcel": ItemData(70, ItemClassification.progression, ["Unique", "Key Items"]),
+    "Item Finder": ItemData(71, ItemClassification.progression, ["Unique", "Key Items"]),
+    "Silph Scope": ItemData(72, ItemClassification.progression, ["Unique", "Key Items"]),
+    "Poke Flute": ItemData(73, ItemClassification.progression, ["Unique", "Key Items"]),
+    "Lift Key": ItemData(74, ItemClassification.progression, ["Unique", "Key Items"]),
+    "Exp. All": ItemData(75, ItemClassification.useful, ["Unique"]),
+    "Old Rod": ItemData(76, ItemClassification.progression_skip_balancing, ["Unique", "Key Items", "Rods"]),
+    "Good Rod": ItemData(77, ItemClassification.progression_skip_balancing, ["Unique", "Key Items", "Rods"]),
+    "Super Rod": ItemData(78, ItemClassification.progression_skip_balancing, ["Unique", "Key Items", "Rods"]),
+    "PP Up": ItemData(79, ItemClassification.filler, ["Consumables"]),
+    "Ether": ItemData(80, ItemClassification.filler, ["Consumables"]),
+    "Max Ether": ItemData(81, ItemClassification.filler, ["Consumables"]),
+    "Elixir": ItemData(82, ItemClassification.filler, ["Consumables"]),
+    "Max Elixir": ItemData(83, ItemClassification.filler, ["Consumables"]),
+    "Tea": ItemData(84, ItemClassification.progression, ["Unique", "Key Items"]),
+    # "Master Sword": ItemData(85, ItemClassification.progression),
+    # "Flute": ItemData(86, ItemClassification.progression),
+    # "Titan's Mitt": ItemData(87, ItemClassification.progression),
+    # "Lamp": ItemData(88, ItemClassification.progression),
```

Comment:
```suggestion
```
This seems.. wrong?

Reply:
These items, while not fully implemented, exist with these item IDs in the base patch and are planned. The names may change, I guess I was the only one that thought the naming scheme I came up with was amusing